Directory.Build.props

```diff
@@ -6,7 +6,6 @@
     <Copyright>$(CopyrightNetFoundation)</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <DebugType>embedded</DebugType>
```

Comment:
Per our conversation - make sure we set DebugType=Portable and `IncludeSymbols` here.

Reply:
Will do. And `DocFileItem` too.